src/Manavo/BootstrapForms/FormBuilder.php

```diff
@@ -203,13 +202,14 @@ protected function checkable($type, $name, $value, $checked, $options)
     public function checkbox(
         $name,
         $value = 1,
-        $label = null,
         $checked = null,
         $options = []
     ) {
         $checkable = parent::checkbox($name, $value, $checked, $options);
 
-        return $this->wrapCheckable($label, 'checkbox', $checkable);
+        return @$options['label'] ?
```

Comment:
Sure, even better! I'll leave it up to you if you want to use `isset` or `array_key_exists` :+1: